src/tests/conftest.py

```diff
@@ -35,34 +35,39 @@ def api_rf() -> APIRequestFactory:
 
 
 @pytest.fixture()
-def api_request(api_rf) -> WSGIRequest:
+def api_request_with_scopes(api_rf) -> Callable:
```

Comment:
Ik vraag me een beetje af of dit nog wel een fixture moet zijn... Het lijkt teveel op een util functie

Reply:
Ik zal er een functie van maken. Ik begon met fixtures aan te passen, tot ik merkte dat dat niet werkte.